guides/common/modules/con_security-content-automation-protocol.adoc

```diff
@@ -3,7 +3,7 @@
 
 {Project} uses the Security Content Automation Protocol (SCAP) standard to define security policies.
 
-SCAP is a framework of several specifications based on XML, such as checklists described in the Extensible Checklist Configuration Description Format (XCCDF) and vulnerabilities described in the Open Vulnerability and Assessment Language (OVAL).
+SCAP is a framework of several specifications based on XML, such as checklists described in the Extensible Checklist Configuration Description Format (XCCDF).
```

Comment:
I disagree. The OVAL component isn't being removed from SCAP specification, is it? @adamruzicka